test/fixtures.js

```diff
@@ -17,6 +17,8 @@ module.exports = nock('https://openshift.redhat.com')
 .reply(200, datareply, headers)
 .get('/broker/rest/cartridges', '*')
 .reply(200, datareply, headers)
+.get('/broker/rest/application/1a')
+.reply(200, datareply, headers)
```

Comment:
@david-martin what do you make of these nock-like fixtures? 
I find them a bit of a pain to debug, but a lot tidier than excessive proxyquiring of the request module